### PR TITLE
Added sudo explanation to install script

### PR DESCRIPTION
**Pull request by arnuschky**
Addresses #1266

### CODE REVIEW

File: configure.sh [hunk 1]
```diff
@@ -257,33 +284,41 @@ function installSlack {
...
     installFedora
   elif [ -f /etc/slackware-version ]; then
+    echo "Found distribution Slack"
     installSlack
   elif grep Raspbian /etc/os-release ; then
-    echo Found Raspberry Pi Raspbian
+    echo "Found distribution Raspberry Pi Raspbian"
     installRaspbian "$@"
   else
+    echo "Found distribution Ubuntu"
```
Comment (hoffmabc) [inline configure.sh]: This is a catch all else statement and we print out Found distribution Ubuntu. We should probably say Found distribution Ubuntu or other.

**Comment by hoffmabc** [comment]
I'm ok with this PR as long as the wording mentioned above is modded. Also if some others could test this out on other OSs I would appreciate an ACK. @gubatron ?

**Comment by gubatron** [comment]
awesome patch! LGTM.

But If you want to make it even better @arnuschky  (as I see you like refactoring) perhaps it'd be cool if we could define all the package dependencies for each OS somewhere in a block of code at the beginning of the script, so they're all easy to access and maintain in one region of code:

```bash
UBUNTU_PACKAGES = "python-pip build-essential rng-tools python-dev libjpeg-dev sqlite3 openssl alien libssl-dev python-virtualenv lintian libjs-jquery"
ARCH_PACKAGES = "python2 python2-pip python2-virtualenv rng-tools libjpeg sqlite3 openssl"
RASPI_PACKAGES = "base-devel curl wget python2 python2-pip rng-tools libjpeg sqlite3 openssl libunistring"
RASPBIAN_PACKAGES = "python-pip build-essential rng-tools alien openssl libssl-dev python-dev libjpeg-dev sqlite3 openssl libssl-dev python-dev libjpeg-dev sqlite3"
PORTAGE_PACKAGES = "dev-lang/python:2.7 dev-python/pip rng-tools gcc jpeg sqlite3 openssl dev-python/virtualenv"
...
```

File: configure.sh [hunk 2]
```diff
@@ -257,33 +284,41 @@ function installSlack {
   ./env/bin/pip install -r requirements.txt
 }
 
+echo "Detecting OS..."
```
Comment (gubatron) [inline configure.sh]: nice, it even ads some swag...

Comment (gubatron) [inline configure.sh]: ROFL

**Pull request merged**